### DISCUSSION TITLE
Increasing node width for long texts by mouse.

### DISCUSSION THREAD

**Discussion by aaa1386** [freeplane/freeplane#1676] [Ideas]
Many times the text that is written or pasted inside a node is a long text.
If by dragging the mouse from the edge of the node, the node can be made wider as desired, like the OneNote software that extends the search box, it would be very good.

![image](https://github.com/freeplane/freeplane/assets/38586259/9909a5b0-85d1-4d98-9693-abe486bd70f2)

**Comment by aaa1386** [comment]
https://github.com/freeplane/freeplane/assets/38586259/b72d8ee9-4aa7-448c-9ccd-04caf96ecc07

**Reply by Illegit** [reply]
If you have not yet tried it: I feel nicely with the command alt+mouse central wheel to enlarge or decrease the width of the node.
Hope it could be helpful.

**Reply by dpolivaev** [reply]
In most cases setting a node's minimal width less than its maximal width preferably in a style definition allows Freeplane to find the best width for a node automatically.

**Reply by Illegit** [reply]
I confirm it is working in my case, I wasn't aware of it. Thanks Dimitry.

**Comment by aaa1386** [comment]
Thanks - two good solutions ✅✅
